src/services/exchangers/nbu-client.ts

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios';
+import { ExchangeClient } from '../exchanger.service';
+import { config } from '../../config';
+import { Currency } from '../../models/currency';
+import logger from '../logger.service';
+import * as _ from 'lodash';
```

Comment:
If you import all the functions in this way, you will not be able to optimise the size of the final application build.
https://webpack.js.org/guides/tree-shaking/ 
So it is better to always specify which functions you need to import.

Reply:
removed